config/processEimage.py

```diff
@@ -0,0 +1 @@
+config.calibrate.photoCal.doSelectUnresolved=False
```

Comment:
Yes.  Like I said in the issue, you'll need a build as of this morning.  I just merged that ticket today.